tests/functional/adapter/test_seeds.py

```diff
@@ -46,6 +48,7 @@
 select /** fetch_timeout_ms(10000) */ /** fetch_mode('streaming') */ * from base where id = 10
 """
 
+
 class TestSeeds():
```

Comment:
Could I ask you to move this class to new directory named `component` so we we will have unit tests in regular directories that correspond to modules its testing, `component` which will have all the tests with the mock you have created and then `e2e` for tests with actual Flink intance. What do you think?

Reply:
done